about.html

```diff
@@ -0,0 +1,166 @@
+            alt="hyundai logo emblem png"
+        /></a>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Job Offer</a></li>
+          <li><a href="#">Help</a></li>
+          <li class="login">
+            <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="paragraphe">
+        <div class="paragraphe_title">
+          <h1>Hyundai Rwanda. Sales and Marketting Place</h1>
+
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo
+            doloremque tempora voluptas maxime? Nulla ex, magni odit nihil
+            aspernatur itaque fugiat numquam odio veritatis placeat perferendis,
+            enim suscipit quas ipsa blanditiis! Et consequatur reiciendis
+            temporibus, animi earum ipsa id distinctio dolor tempora mollitia,
+            similique?
+          </p>
+          <small>
+            If you want a car, please contact using this email
+            www.hyundai.rw</small
+          >
+        </div>
+      </section>
+
+      <section id="courses" class="my-team">
+        <p>VISIT OUR BRAND ALL OVER THE WORLD</p>
+        <hr />
+        <div class="teamList">
+          <div class="listPara">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod,
+              corrupti doloribus. Quam temporibus cum culpa fugiat. Quae aut
+              repudiandae magnam, ullam fuga sapiente, laboriosam expedita
+              voluptas natus explicabo reprehenderit esse. Quae, ipsum quaerat
+              modi facere harum nam recusandae voluptate nesciunt consectetur
+              amet enim corrupti delectus accusantium quasi dicta perferendis
+              veniam.repudiandae magnam, ullam fuga sapiente, laboriosam
+              expedita voluptas natus explicabo reprehenderit esse. Quae, ipsum
+              quaerat modi facere harum nam recusandae voluptate nesciunt
+              consectetur amet enim corrupti delectus accusantium quasi dicta
+              perferendis veniam.
+            </p>
+          </div>
+          <div class="brand">
+            <a
+              href="https://www.freepnglogos.com/pics/hyundai-logo"
+              title="Image from freepnglogos.com"
+              ><img
+                src="https://www.freepnglogos.com/uploads/hyundai-logo-0.png"
+                alt="hyundai logo emblem png"
+            /></a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="marks">
+        <p>SEE OUR TEAM WORKING STRATEGY</p>
+        <hr />
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
```

Comment:
- [ ] One of the project requirements is to follow the design template in Behance site given to you. It would be nice to follow the design template for this section. For this section, you should have two images with centered text overlapping the image. Kindly make this section look like the image below 

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/153725616-9dbd113d-cf9b-4983-acde-6e972548e99f.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/153725581-41616839-8035-45c4-b17b-d4a2b9d142d1.png)